src/auro-datetime.js

```diff
@@ -76,97 +64,84 @@ class AuroDatetime extends LitElement {
-  static get styles() {
-    return css`
-      ${styleCss}
-    `;
-  }
-
   /**
-   * @private Internal function generate standard date string
+   * @private Internal function generate date string
    * @returns {string} - date string
    */
   humanDate() {
-    const newDate = new Date();
+    let newDate = new Date();
 
-    return newDate.toLocaleString('en-us', this.dateTemplate);
-  }
+    if (this.utc) {
+      this.dateTemplate.timeZone = 'UTC';
+      newDate = new Date(this.utc);
 
-  /**
-   * @private Internal function generate numeric date string
-   * @returns {string} - date string
-   */
-  numericDate() {
-    this.dateTemplate.month = 'numeric';
-    Reflect.deleteProperty(this.dateTemplate, 'weekday');
-    const newDate = new Date();
+      return newDate.toLocaleString('en-us', this.dateTemplate);
+    } else if (this.setDate) {
+      newDate = new Date(this.setDate);
+
+      return newDate.toLocaleString('en-us', this.dateTemplate);
+    }
 
     return newDate.toLocaleString('en-us', this.dateTemplate);
   }
 
   /**
-   * @private Internal function generate month date string
+   * @private Internal function to determine new Date object based on input type
    * @returns {string} - date string
    */
-  humanDateMonth() {
-    this.template.month = this.month;
-    const newDate = new Date();
+  humanDateConversion() {
+    let newDate = new Date();
 
-    return newDate.toLocaleString('en-us', this.template);
-  }
+    if (this.utc) {
+      this.template.timeZone = 'UTC';
+      newDate = new Date(this.utc);
+    } else if (this.setDate) {
+      newDate = new Date(this.setDate);
+    }
 
-  /**
-   * @private Internal function generate year date string
-   * @returns {string} - date string
-   */
-  humanDateYear() {
-    this.template.year = 'numeric';
-    const newDate = new Date();
+    switch (this.type) {
+      case 'day':
+        this.template.day = "numeric";
 
-    return newDate.toLocaleString('en-us', this.template);
-  }
+        return newDate.toLocaleString('en-us', this.template);
+      case 'month':
+        this.template.month = this.month;
 
-  /**
-   * @private Internal function generate weekday date string
-   * @returns {string} - date string
-   */
-  humanDateWeekday() {
-    this.template.weekday = this.weekday;
-    const newDate = new Date();
+        return newDate.toLocaleString('en-us', this.template);
+      case 'year':
+        this.template.year = 'numeric';
 
-    return newDate.toLocaleString('en-us', this.template);
-  }
+        return newDate.toLocaleString('en-us', this.template);
+      case 'weekday':
+        this.template.weekday = this.weekday;
 
-  /**
-   * @private Internal function generate day date string
-   * @returns {string} - date string
-   */
-  humanDateDay() {
-    this.template.day = "numeric";
-    const newDate = new Date();
+        return newDate.toLocaleString('en-us', this.template);
+
+      default: newDate.toLocaleString('en-us', this.template);
```

Comment:
Did you mean to not return here? And if we return here, do we still need to have the `return null` fallback?

Reply:
If possible, it would be cleaner to do something like:
```js
switch (this.type) {
  case 'day':
    this.template.day = 'numeric';
  case 'month':
    this.template.month = this.month;
// etc

return newDate.toLocaleString('en-us', this.template);
```

Instead of returning from each case.

Reply:
look at you ... making me better.